node-repository/src/test/java/com/yahoo/vespa/hosted/provision/restapi/v2/responses/node4-after-changes.json

```diff
@@ -39,6 +39,7 @@
   "failCount": 0,
   "hardwareFailure": true,
   "hardwareFailureType": "memory_mcelog",
+  "hardwareFailureDescription": "memory_mcelog",
```

Comment:
The added test doesn't test what I want it to test: Say a node has the hardwareFailure field set in node repo. Verify the JSON response contains that value for the hardwareFailureDescription field.